package.json

```diff
@@ -30,7 +32,6 @@
     "@types/react-dom": "^18.3.1",
     "@vitejs/plugin-react": "^4.3.4",
     "eslint": "^9.15.0",
-    "eslint-plugin-react-hooks": "^5.0.0",
```

Comment:
Eslint plugins are very helpful and can guide us know the best practices and if we are doing anything wrong. If we installed the eslint vs code extention, we'll get error warnning instantly related to things that we might be doing wrong. For the react hooks plugin, it will help us follow the rules of hooks which are very crucial and not following them might lead to unexpected bugs.

I suggest reading about these rules here.
https://react.dev/warnings/invalid-hook-call-warning

I suggest watching this video as well about eslint.
https://youtu.be/K9vCqbZ9Wec?si=hBDQ9itdT-FENuVa